src/igvfd/schemas/treatment.json

```diff
@@ -0,0 +1,206 @@
+{
+    "title": "Treatment",
+    "id": "/profiles/treatment.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "amount",
+        "amount_units",
+        "treatment_term_id",
+        "treatment_term_name",
+        "treatment_type"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/aliases"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/notes"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        }
+    ],
+    "dependentSchemas": {
+        "amount": {
+            "required": [
+                "amount_units"
+            ],
+            "comment": "amount_units must be specified if amount is specified"
+        },
+        "duration": {
+            "required": [
+                "duration_units"
+            ],
+            "comment": "duration_units must be specified if duration is specified"
```

Comment:
I think this dependency means that one can specify duration_units, but not duration? I think you need to add the other "direction". If the duration is specified you have to have units, and if you have units you have to have duration.

Reply:
same for post_treatment_units and post_treatment_time

Reply:
@meenakshikagda you can use the dependancies in Ian's PR for Organiod as an example